CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ should change the heading of the (upcoming) version to include a major version b
 ## @rjsf/core
 - Added `ref` definition to `ThemeProps` fixing [#2135](https://github.com/rjsf-team/react-jsonschema-form/issues/2135)
 - Updated the `onChange` handler in `Form` to use the new `preventDuplicates` mode of `mergeObjects()` when merging `extraErrors` when live validation is off, fixing [#3169](https://github.com/rjsf-team/react-jsonschema-form/issues/3169)
+- Fix isObject to correctly identify 'Date' as not mergable with Object default value.
```

Comment:
This actually belongs in the `@rjsf/utils` section below. I'll approve once it is moved.

Reply:
🤦‍♂️sorry - I should've seen that.